google-sheets/.env

```diff
@@ -0,0 +1,13 @@
+# description: The document ID for the Google Sheet you want to log into (from its URL: https://docs.google.com/spreadsheets/d/[ID]/edit)
+# format: text
+# required: true
+SHEETS_DOC_ID=
+
+# description: The spreadsheet name to append to within your Google Sheets document
+# format: text
+# required: true
+SHEETS_SHEET_NAME=Sheet1
+
+# description: The path to the service account authentication file
+# configurable: false
+SHEETS_SERVICE_ACCOUNT_JSON=./assets/auth.json
```

Comment:
Should this file be here?

Reply:
Interesting... I'm actually not sure how the `google-sheets` `.env` ended up here, and the `zapier` `.env` ended up getting erased. One moment, let me recreate the correct file.

Reply:
Ah, looks like I accidentally ran `git add` for the wrong file on the wrong branch.

Reply:
I amended the commit to have the correct `.env` file for this function template.